lib/html2js.js

```diff
@@ -49,14 +53,18 @@ var createHtml2JsPreprocessor = function (logger, basePath, config) {
     }
 
     var tpl
-    if (moduleName) {
-      tpl = util.format(SINGLE_MODULE_TPL, moduleName, moduleName, htmlPath, escapeContent(content))
+    if (angular === 2) {
```

Comment:
Our linting will not allow that, but you can always do `angular === 2 || angular === '2'`